circuitx/effects/build.gradle.kts

```diff
@@ -1,9 +1,7 @@
-// Copyright (C) 2024 Slack Technologies, LLC
+// Copyright (C) 2023 Slack Technologies, LLC
```

Comment:
Wrong year? Seems to have changed the year back in a few files.

Reply:
Yeah I think spotless got confused on a couple build files that changed imports

```suggestion
// Copyright (C) 2024 Slack Technologies, LLC
```